### DISCUSSION TITLE
Lightbulb.exe prevents windows 10 from shutting down

### DISCUSSION THREAD

**Discussion by MillionLights** [Tyrrrz/LightBulb#341] [Q&A] [answered]
I'm not sure if the issue is on my end or caused by Lightbulb but when I try to turn off my PC,
Windows gives me a message that a program is preventing Windows from shutting down.

The Eventviewer log tells me it's caused by Lightbulb.exe
![image](https://github.com/user-attachments/assets/92583c72-46da-40e3-b571-28931fe16146)

I know a way around it but wanted to share my experience in case it is a bug.

**Comment by Tyrrrz** [comment]
Hm, what does it actually look like? Does it show up in that "These apps are preventing Windows from shutting down" window that pops up, or something else? Trying to see if it's something I experienced or not

**Reply by MillionLights** [reply]
Yes. Windows logs out but stops with the message that "This app is preventing shutdown".
But because there is no icon I had to go in Eventviewer to figure out what app it is.
![image](https://github.com/user-attachments/assets/c1abb28f-6f74-4a04-a97c-c1fb7bff678b)

**Reply by Tyrrrz** [reply]
I see. Just to confirm, can you do the following:

- Exit LightBulb before shutting down to make sure it's definitely the one that's delaying the shutdown (and not waiting on another process that does).
- Reset LightBulb to default settings and see if the behavior changes in any way. You can backup the settings file to restore it later: `%appdata%/LightBulb/Settings.json`

**Reply by MillionLights** [reply, marked as answer]
Interesting.

I closed lightbulb and tried to shutdown the PC.

Same error BUT eventviewer didnt log anything.
tried again, same error but that time eventiewer DID log an error.
But a different Program this time.

repeated that 2 more times and it just jumped to a different program every time I manually closed the one mentioned before.

Its safe to say that it is an issue with my Windows installation and not related to Lightbulb.

Thank you for your help and thoughts on the issue!
